postgres/tests/unit.tftest.hcl

```diff
@@ -312,21 +306,11 @@ run "aws_db_instance_unit_test_set_to_non_defaults" {
     error_message = "Should be: 20"
   }
 
-  assert {
-    condition     = aws_db_instance.default.iops == 3000
-    error_message = "Should be: 20"
-  }
-
```

Comment:
Won't pass with the old config, will put it back when we revert the changes in this pull request.